src/getHumanAge.test.js

```diff
@@ -7,4 +7,37 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return 0, if animalAge less than first', () => {
+    const result = getHumanAge(14, 14);
+
+    expect(result).toEqual([0, 0]);
+  });
+
+  test(`should return 1,
+      if animalAge less than (first + second) and more than first`, () => {
+    const result = getHumanAge(23, 23);
+
+    expect(result).toEqual([1, 1]);
+  });
+
+  test(`should return 2, if animalAge equal (first + second)`, () => {
+    const result = getHumanAge(27, 27);
+
+    expect(result).toEqual([2, 2]);
+  });
+
+  test(`should return correct value,
```

Comment:
The test description should specify that `catAge` and `dogAge` are more than 24, which corresponds to more than 2 human years according to the task description.